src/components/Navigation.jsx

```diff
@@ -240,6 +240,7 @@ export const navigation = [
     title: 'App States',
     links: [
       { title: 'Notebooks', href: '/states/books' },
+      { title: 'editingNote', href: '/states/editing-note' },
```

Comment:
Personally, I think it would be better if the sidebar had a generally understandable and readable name.
```suggestion
      { title: 'Currently open note', href: '/states/editing-note' },
```

Reply:
yep, it should be 'Currently opened note'